scripts/globals/weaponskills.lua

```diff
@@ -959,3 +959,61 @@ function takeWeaponskillDamage(defender, attacker, params, finaldmg, slot, tpHit
 
     return finaldmg;
 end
+
+function applyAftermathEffect(player, tp, spower_1, spower_2, spower_3)
+    spower_1 = spower_1 or 1
+    spower_2 = spower_2 or 1
+    spower_3 = spower_3 or 1
+
+    -- AFTERMATH LEVEL 1
+
+    if ((tp >= 1000) and (tp <= 1100)) then
```

Comment:
Ok, so if they're linear within their ranges, then is it a bug that level 2 increments by 4 at every level except the last, where it increments by 3?